skorch/net.py

```diff
@@ -34,6 +34,16 @@
 from skorch.utils import to_tensor
 
 
+PYTORCH_COMPONENTS = {'criterion', 'module', 'optimizer'}
```

Comment:
Private for now?

```suggestion
_PYTORCH_COMPONENTS = {'criterion', 'module', 'optimizer'}
```

Reply:
Done. We can make it public later if there is a need.